### ISSUE TITLE
Doesn't work with `export default` js syntax

### ISSUE THREAD

**Issue by esetnik** [kkoomen/vim-doge#86]
**Describe the bug**

// Test.tsx
```tsx
import React from 'react';

interface TestProps {
  fieldName: string;
}

export default function Test({
  fieldName,
}: TestProps) {
  return <label htmlFor={fieldName}>credit card</label>;
}
```

**Settings**
None

**To Reproduce**
1. Create a tsx file like above
2. Bring cursor to `Test`
3. \<leader\>f
4. Nothing happens
5. Delete export default
6. Bring cursor to `Test`
7. \<leader\>f
8. Correct documentation block is generated

**Expected behavior**
Should behave the same way whether or not `export default` is used.

**Screenshots**

**Out of `vim --version`**
```
❯ nvim -v
NVIM v0.4.3
Build type: Release
LuaJIT 2.0.5
Compilation: /usr/local/Homebrew/Library/Homebrew/shims/mac/super/clang -U_FORTIFY_SOURCE -D_FORTIFY_SOURCE=1 -DNDEBUG -DMIN_LOG_LEVEL=3 -Wall -Wextra -pedantic -Wno-unused-parameter -Wstrict-prototypes -std=gnu99 -Wshadow -Wconversion -Wmissing-prototypes -Wimplicit-fallthrough -Wvla -fstack-protector-strong -fdiagnostics-color=auto -DINCLUDE_GENERATED_DECLARATIONS -D_GNU_SOURCE -DNVIM_MSGPACK_HAS_FLOAT32 -DNVIM_UNIBI_HAS_VAR_FROM -I/tmp/neovim-20191107-85913-1wbgqz6/neovim-0.4.3/build/config -I/tmp/neovim-20191107-85913-1wbgqz6/neovim-0.4.3/src -I/usr/local/include -I/tmp/neovim-20191107-85913-1wbgqz6/neovim-0.4.3/deps-build/include -I/usr/local/opt/gettext/include -I/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX10.15.sdk/usr/include -I/tmp/neovim-20191107-85913-1wbgqz6/neovim-0.4.3/build/src/nvim/auto -I/tmp/neovim-20191107-85913-1wbgqz6/neovim-0.4.3/build/include
Compiled by brew@Catalina.local

Features: +acl +iconv +tui
See ":help feature-compile"

   system vimrc file: "$VIM/sysinit.vim"
  fall-back for $VIM: "/usr/local/Cellar/neovim/0.4.3/share/nvim"

Run :checkhealth for more info
```

**Comment by kkoomen** [comment]
This bug has been fixed and released in [v2.4.1](https://github.com/kkoomen/vim-doge/tree/v2.4.1).

Feel free to submit any new issues if you experience any unwanted behavior in the future. Thanks for your contribution.

**Comment by esetnik** [comment]
Thanks for the quick fix. I confirmed it's working well now.
